<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KirjausApuriDialog</class>
 <widget class="QDialog" name="KirjausApuriDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kirjausapuri</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="alvVaaraKuva">
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../pic/pic.qrc">:/pic/varoitus.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="alvVaaraTeksti">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Arvonlisäveroilmoitus on jo annettu&lt;br/&gt;&lt;/span&gt;Tälle päivämäärälle ei voi enää tehdä verollista kirjausta!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabBar" name="valintaTab" native="true"/>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="tiliLabel">
       <property name="text">
        <string>&amp;Debet-tili</string>
       </property>
       <property name="buddy">
        <cstring>tiliEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="TilinvalintaLine" name="tiliEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="eraLabel">
       <property name="text">
        <string>Tase-&amp;erä</string>
       </property>
       <property name="buddy">
        <cstring>taseEraCombo</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="taseEraCombo"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>&amp;Määrä</string>
       </property>
       <property name="buddy">
        <cstring>euroSpin</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="euroSpin">
       <property name="suffix">
        <string>€</string>
       </property>
       <property name="minimum">
        <double>-9999999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>9999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="nettoLabel">
       <property name="text">
        <string>Ve&amp;roton</string>
       </property>
       <property name="buddy">
        <cstring>nettoSpin</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="nettoSpin">
       <property name="suffix">
        <string>€</string>
       </property>
       <property name="minimum">
        <double>-9999999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>9999999.000000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="vastatiliLabel">
       <property name="text">
        <string>K&amp;redit-tili</string>
       </property>
       <property name="buddy">
        <cstring>vastatiliEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="TilinvalintaLine" name="vastatiliEdit"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="kohdennusLabel">
       <property name="text">
        <string>K&amp;ohdennus</string>
       </property>
       <property name="buddy">
        <cstring>kohdennusCombo</cstring>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QComboBox" name="kohdennusCombo"/>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Vienti&amp;pvm</string>
       </property>
       <property name="buddy">
        <cstring>pvmDate</cstring>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QDateEdit" name="pvmDate"/>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>&amp;Selite</string>
       </property>
       <property name="buddy">
        <cstring>seliteEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLineEdit" name="seliteEdit"/>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="alvlajiLabel">
       <property name="text">
        <string>Alv-&amp;laji</string>
       </property>
       <property name="buddy">
        <cstring>alvCombo</cstring>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QComboBox" name="alvCombo"/>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="alvprossaLabel">
       <property name="text">
        <string>Alv-&amp;prosentti</string>
       </property>
       <property name="buddy">
        <cstring>alvSpin</cstring>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QSpinBox" name="alvSpin">
       <property name="suffix">
        <string>%</string>
       </property>
       <property name="prefix">
        <string/>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="vastaTaseEraLabel">
       <property name="text">
        <string>Tase-erä</string>
       </property>
       <property name="buddy">
        <cstring>vastaTaseEraCombo</cstring>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QComboBox" name="vastaTaseEraCombo"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="poistoLabel">
       <property name="text">
        <string>Poisto&amp;aika</string>
       </property>
       <property name="buddy">
        <cstring>poistoSpin</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="poistoSpin">
       <property name="suffix">
        <string> vuotta</string>
       </property>
       <property name="maximum">
        <number>1200</number>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QCheckBox" name="yhdistaCheck">
       <property name="text">
        <string>&amp;Yhdistä aiempaan vientiin</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="ehdotusView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="vaihdaNappi">
       <property name="text">
        <string>&amp;Vaihda tilit</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TilinvalintaLine</class>
   <extends>QLineEdit</extends>
   <header>db/tilinvalintaline.h</header>
  </customwidget>
  <customwidget>
   <class>QTabBar</class>
   <extends>QWidget</extends>
   <header>QTabBar</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tiliEdit</tabstop>
  <tabstop>taseEraCombo</tabstop>
  <tabstop>poistoSpin</tabstop>
  <tabstop>euroSpin</tabstop>
  <tabstop>nettoSpin</tabstop>
  <tabstop>vastatiliEdit</tabstop>
  <tabstop>vastaTaseEraCombo</tabstop>
  <tabstop>kohdennusCombo</tabstop>
  <tabstop>pvmDate</tabstop>
  <tabstop>seliteEdit</tabstop>
  <tabstop>alvCombo</tabstop>
  <tabstop>alvSpin</tabstop>
  <tabstop>vaihdaNappi</tabstop>
  <tabstop>ehdotusView</tabstop>
 </tabstops>
 <resources>
  <include location="../pic/pic.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>KirjausApuriDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>587</x>
     <y>625</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>KirjausApuriDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>617</x>
     <y>625</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
